packages/mux-player/src/template.ts

```diff
@@ -40,11 +40,13 @@ export const template = (props: MuxTemplateProps) => html`
       debug="${props.debug}"
       prefer-mse="${props.preferMse}"
       start-time="${props.startTime != null ? props.startTime : false}"
-      poster="${!!props.poster
-        ? props.poster
-        : props.playbackId
-        ? getPosterURLFromPlaybackId(props.playbackId)
-        : false}"
+      poster="${cache(
```

Comment:
could this instead be handled by making mux-video smarter about updating its internal state only if the value changes (likely for many attributes)?

Reply:
I'll try it out